### DISCUSSION TITLE
How to style a menu item differently - if it is on the same level as the rest?

### DISCUSSION THREAD

**Discussion by andreimoment** [vadikom/smartmenus#160] [Q&A]
If all the items are on the same level - would it be possible to style one of them differently?

How would I approach such a scenario?

Example:

* Offices (first level)
  * New York  (second level) 
  * London  (second level) 
  * Beijing  (second level) 
  * Our Global Presence (second level - different styling)

**Comment by vadikom** [comment]
You can just add a class to any item you like:
```html
<li><a class="my-item" href="#">My item</a></li>
```
and add some CSS for it in some of your stylesheets:
```css
.my-item {
    color: red !important;
}
```
Note: You might need to use the `!important` declaration to override some of the default styles in the theme you use.

**Reply by andreimoment** [reply]
Thank you!
